bin/cli.js

```diff
@@ -78,6 +79,13 @@ program
   .action(config.wrap(program, init.cmd));
 
 program
+  .command("env")
+  .description("Output configured env variables")
+  .option("-s --site-id [id]", "Fetch from site with <id>")
+  .option("-f --file [filename]", "Save to file called <filename>")
```

Comment:
I think cutting this is better. It simplifies the code. You can just use a redirect b/c this is a cli